### ISSUE TITLE
Annotate theoretical number of peptides in PeptideIndexer

### ISSUE THREAD

**Issue by timosachsenberg** [OpenMS/OpenMS#4034]
e.g., for emPAI

**Comment by jpfeuffer** [comment]
Yes that is very helpful but I am not sure if PeptideIndexer is the right place.
We then depend on the correct settings of one of the first tools in a pipeline for one of the last tools (quant.). This will lead to a lot of errors and questions.
I'd rather see a nice API for other tools to use
so they can re-annotate proteins with such kind of information, given the initial database.
I would also be very interested in StringViews of the missing peptides (i.e. start-end positions) so that you could differentiate in their importance (e.g. Proteotypicity).

**Comment by timosachsenberg** [comment]
Enzyme Information is already used in PI so I thought it’s fine to add there
